src/components/RatingsChart/RatingsChart.tsx

```diff
@@ -613,7 +733,7 @@ export class RatingsChart extends React.PureComponent<RatingsChartProperties, an
             }
         }
 
-        /* Plot */
+        /* Plot graph.... */
```

Comment:
And if you're going to use ellipsis, it's exactly 3 dots, not 4! :) (Note: I might be feeling a bit pedantic this morning.)

BTW I'm a big overuser of ellipsis myself, but I'm trying to get better. As a result, you get to get better with me!